libraries/AP_Compass/AP_Compass_config.h

```diff
@@ -88,6 +88,10 @@
 #define AP_COMPASS_ICM20948_ENABLED AP_COMPASS_I2C_BACKEND_DEFAULT_ENABLED
 #endif
 
+#ifndef AP_COMPASS_IIS2MDC_ENABLED
+#define AP_COMPASS_IIS2MDC_ENABLED 0
```

Comment:
if we change this to AP_SIM_ENABLED instead of 0 then that will ensure it is at least build tested